connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -0,0 +1,217 @@
+          type: array
+          description: HTTP requests sent to the source API, and their responses
+          items:
+            type: object
+            required:
+              - request
+              - response
+            properties:
+              request:
+                type: object
+                description: The HTTP request sent to the source API
+                required:
+                  - url
+                  - headers
+                properties:
+                  url:
+                    type: string
+                    description: URL that the request was sent to
+                  body:
+                    type: object
+                    description: The body of the HTTP request, if present
+                  headers:
+                    type: object
+                    description: The headers of the HTTP request
+              response:
+                type: object
+                description: The response from the source API
+                required:
+                  - status
+                  - headers
+                properties:
+                  status:
+                    type: integer
+                    description: The status of the response
+                  body:
+                    type: object
+                    description: The body of the HTTP response, if present
+                  headers:
+                    type: object
+                    description: The headers of the HTTP response
+    StreamReadRequestBody:
+      type: object
+      required:
+        - definition
+        - stream
+      properties:
+        definition:
+          $ref: "#/components/schemas/ConnectorDefinitionBody"
+          description: The config-based connector definition contents
+        stream:
+          type: string
+          description: Name of the stream to read
+    ConnectorDefinitionBody:
+      $ref: ../../../../airbyte-cdk/python/airbyte_cdk/sources/declarative/config_component_schema.json
+    StreamsListRequestBody:
+      type: object
+      required:
+        - definition
+      properties:
+        definition:
+          $ref: "#/components/schemas/ConnectorDefinitionBody"
+          description: The config-based connector definition contents
+    StreamsListRead:
+      type: object
+      required:
+        - streams
+      properties:
+        streams:
+          type: array
+          items:
+            type: object
+            description: The stream names present in the connector definition
+            required:
+              - name
+              - url
+            properties:
+              name:
+                type: string
+                description: The name of the stream
+              url:
+                type: string
+                format: uri
+                description: The URL to which read requests will be made for this stream
+
+    # The following exception structs were copied from airbyte-api/src/main/openapi/config.yaml
```

Comment:
I just copied these exception structs over since I was not super opinionated on exactly how the server should communicate exceptions. Feel free to suggest or push any changes to these if desired

Reply:
seems reasonable